sdk/core/Azure.Core/src/Pipeline/Policies/BufferResponsePolicy.cs

```diff
@@ -13,7 +13,7 @@ protected BufferResponsePolicy()
         {
         }
 
-        public static HttpPipelinePolicy Singleton { get; set; } = new BufferResponsePolicy();
+        public static HttpPipelinePolicy Shared { get; set; } = new BufferResponsePolicy();
```

Comment:
I might be missing context here and that is why idk what drive the name change, but could you explain more why?

Reply:
We had some policies that used Singleton others - Shared, we are aligning on `Shared`

Reply:
Thanks!